src/main/java/frc/robot/OI.java

```diff
@@ -1,11 +1,59 @@
 package frc.robot;
 
+import edu.wpi.first.networktables.EntryListenerFlags;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.utils.TrigonXboxController;
+
 /**
  * This class is the glue that binds the controls on the physical operator
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
+    private static final int kDriverPort = 0;
+    private static final int kOperatorPort = 1;
+    private TrigonXboxController driverXbox;
+    private TrigonXboxController operatorXbox;
     public OI() {
+        createOperatorCommands();
+        driverXbox = new TrigonXboxController(kDriverPort);
+        operatorXbox = new TrigonXboxController(kOperatorPort);
+        NetworkTableEntry changeSettingsEntry = SmartDashboard.getEntry("isHillelSettings");
+        changeSettingsEntry.setBoolean(true);
+        // Listens for operator settings change. If it changes while the robot is disabled, the OI updates the button bindings.
+        changeSettingsEntry.addListener(entryNotification -> {
+            if (DriverStation.getInstance().isDisabled()) {
+                CommandScheduler.getInstance().clearButtons();
```

Comment:
CommandScheduler.getInstance().clearButtons();
also clears yoav's setings... find a way to keep them or call **setDriverCommands()** (bind method like the other set methods) again after it..